_includes/usage/cf-grid.md

```diff
@@ -36,16 +36,6 @@ Read on for more details on the variables and mixins this component provides.
   or your compiled Capital Framework CSS will
   not work perfectly in older browsers.
 
-## Table of contents
```

Comment:
@Scotchester did you add this here? It seems the usage.md this is copied from doesn't include a ToC.

Reply:
I added it back in October https://github.com/cfpb/capital-framework/commit/ba782f45243b0e89b3e40916ff289b79c23070fc#diff-03a6d671db586ccdd7a5dd6650e1298c

I think it should stay, particularly since we have variables in a TOC elsewhere. The more similarly formatted the components usage.md files are, the better.

Reply:
@anselmbradford Weird, why is it not included in the build when I run it but is when Scott runs it? Could you pull this branch down and give `npm install && gulp build` a try?